lnbits/app.py

```diff
@@ -270,6 +270,25 @@ def register_routes(app: FastAPI) -> None:
             )
 
 
+def register_custom_extensions_path():
+    if settings.has_default_extension_path:
+        return
+    default_ext_path = "lnbits/extensions"
+    if os.path.isdir(default_ext_path) and len(os.listdir(default_ext_path)) != 0:
+        logger.warning(
+            "You are using a custom extensions path, "
+            + "but the default extensions directory is not empty. "
+            + f"Please clean-up the '{default_ext_path}' directory."
+        )
+        logger.warning(
+            f"You can move the existing '{default_ext_path}' directory to: "
+            + f" '{settings.lnbits_extensions_path}/extensions'"
```

Comment:
should use `os.path.join()` instead of a string concatenation.

Reply:
its just a log message